zcash_client_backend/src/data_api.rs

```diff
@@ -239,19 +239,19 @@ pub struct AccountBalance {
     /// possible operation on a transparent balance is to shield it, it is possible to create a
     /// zero-conf transaction to perform that shielding, and the resulting shielded notes will be
     /// subject to normal confirmation rules.
```

Comment:
Please also update the comment. It should just say "The value of unspent transparent outputs belonging to the account."